backend/src/prompts/templates/validator.j2

```diff
@@ -4,6 +4,8 @@ Your entire purpose is to return a boolean value to indicate if the answer has f
 
 You will be passed a task and an answer. You need to determine if the answer is correct or not.
 
+Be lenient - if the answer looks reasonably right (isn't a million miles off) then return True

```

Comment:
I think the phrase "isn't a million miles off" is a bit misleading for the LLM

Reply:
Yeah in hindsight I totally agree, removed